mdata/cache/ccache.go

```diff
@@ -131,7 +141,9 @@ func (c *CCache) evict(target *accnt.EvictTarget) {
 	}
 }
 
-func (c *CCache) Search(metric string, from, until uint32) *CCSearchResult {
+func (c *CCache) Search(ctx context.Context, metric string, from, until uint32) *CCSearchResult {
+	ctx, span := tracing.NewSpan(ctx, c.tracer, "CCache.Search")
```

Comment:
I'm surprised the compiler doesn't complain about this `ctx` being unused, I assume that's probably because it's a recycled name, but anyway it's unused